main.rb

```diff
@@ -31,6 +34,7 @@ def perform_action(option)
     end_program(option)
     exit
   end
+  @storage.save_data(@app)
```

Comment:
- [ ] Good job so far 👍 Kindly store your data on exit of your app, hence your `@storage.save_data(@app)` should come before the `exit`, just under the `end_program(option)` so that your data will be store when a user selects option 7.